tests/common/dualtor/mux_simulator_control.py

```diff
@@ -322,3 +327,51 @@ def _drop_helper(interface_name):
 
     yield _drop_helper
     set_output(tmp_list[0], [UPPER_TOR, LOWER_TOR])
+
+@pytest.fixture
+def simulator_flap_counter(url):
+    """
+    A function level fixture to retrieve mux simulator flap counter for a given interface
+    """
+    def _simulator_flap_counter(interface_name):
+        server_url = url(interface_name, FLAP_COUNTER)
+        counter = _get(server_url)
+        assert(counter and len(counter) == 1)
+        return counter.values()[0]
```

Comment:
To avoid a `TypeError`

```suggestion
        return list(counter.values())[0]
```

Reply:
Thanks. Updated.